app-config.yaml

```diff
@@ -206,6 +206,13 @@ scaffolder:
     api:
       token:
         $env: AZURE_TOKEN
+  bitbucket:
```

Comment:
This change needs also to be added to the config schema for the scaffolder backend. `config.d.ts` But it's not required yet.

Reply:
> Will this feature be in Thursday's release?

Yes :)